java/messages/src/main/java/com/nedap/retail/messages/Client.java

```diff
@@ -141,8 +141,13 @@ public String captureErpStock(final Stock stock) {
      * @param id ID of stock
```

Comment:
Please add documentation for the new parameter and add an extra method for backwards compatibility.